### DISCUSSION TITLE
fatal error: MP3DecoderMAD.h: No such file or directory

### DISCUSSION THREAD

**Discussion by korova99** [pschatzmann/arduino-audio-tools#1807] [Q&A] [answered]
While compiling the 'streams-url_mp3_mad-i2s.ino' file included in the example,
I meet ....

fatal error: MP3DecoderMAD.h: No such file or directory

I've confirmed that the file exists in the directory using the file explorer, but I'm getting this error during compilation. What could be the cause?

**Comment by pschatzmann** [comment, marked as answer]
Read the codec wiki!

**Reply by korova99** [reply]
Resolve by installing additional codecs !
